api/database/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM postgres:alpine
+WORKDIR /docker-entrypoint-initdb.d
```

Comment:
Most of this is unnecessary:
- Setting the WORKDIR doesn't matter since all of the destination paths below it use absolute paths.
- The 2 ADDs repeat what COPY already did.
- The chmod seems to be unnecessary.
- postgres:alpine already exposes port 5432.

If we take all of that away, we're left with `COPY *.sql /docker-entrypoint-initdb.d/`, which is already being done via (and actually being overridden by) the Docker volume in docker-compose.yml `./api/database:/docker-entrypoint-initdb.d`. So this entire Dockerfile can be deleted, and the line `build: ./api/database` in docker-compose.yml can be replaced with `image: "postgres:alpine"`.

And ideally we'd move all of the .sql files in api/database/ into a separate api/database/docker-entrypoint-initdb.d/ folder so we're not sharing more files with the db container than we mean to.

Reply:
removed unnecessary code. updated docker compose
have not moved " all of the .sql files in api/database/ into a separate api/database/docker-entrypoint-initdb.d/ folder" yet.

Reply:
Per the middle of my comment, you can actually completely delete api/database/Dockerfile if you change one line of docker-compose.yml.

Reply:
updated. 
Removed Dockerfile of the database. 
Moved all of the .sql files in api/database/ into api/database/docker-entrypoint-initdb.d/ folder